File: ray-tracer/scene/Scene.cs
using ray_tracer.common;

namespace ray_tracer.scene
{
    public class Scene
    {
        public readonly Thing[] things;
        public readonly Light[] lights;
        public readonly Camera camera;

        public Scene(Thing[] things, Light[] lights, Camera camera)
        {
            this.things = things;
            this.lights = lights;
            this.camera = camera;
        }
    }
}

File: java-app/src/pl/mikolajczuk/kamil/raytracing/lib/scene/Scene.java
package pl.mikolajczuk.kamil.raytracing.lib.scene;

import pl.mikolajczuk.kamil.raytracing.lib.common.*;

public class Scene {
    public final Thing[] things;
    public final Light[] lights;
    public final Camera camera;

    public Scene(Thing[] things, Light[] lights, Camera camera) {
        this.things = things;
        this.lights = lights;
        this.camera = camera;
    }
}
